app-frontend/src/app/components/projects/layerSplitModal/index.html

```diff
@@ -0,0 +1,169 @@
+          <div class="form-group all-in-one no-margin">
+            <select
+              class="form-control"
+              name="frequency"
+              ng-model="$ctrl.layerSplitBuffer.period"
+            >
+              <option
+                value="{{frequency.value}}"
+                ng-repeat="frequency in $ctrl.frequencies track by $index"
+              >
+                {{frequency.name}}
+              </option>
+            </select>
+          </div>
+          <p>
+            <i>One new layer will be created for <span class="font-700">every {{ $ctrl.layerSplitBuffer.period | lowercase }}</span> imagery is found.</i>
+          </p>
+          <div class="flex-display">
+            <h5 class="modal-content-header-thin">Start date</h5>
+            <h5 class="modal-content-header-thin">End date</h5>
+          </div>
+          <div
+            class="form-group all-in-one"
+            ng-class="{'no-margin': !$ctrl.isValidDateRange}"
+          >
+            <input class="form-control input-cursor-pointer"
+                   id="rangeStart"
+                   name="rangeStart"
+                   type="text"
+                   ng-value="$ctrl.startDateDisplay"
+                   ng-click="$ctrl.onDateChange('rangeStart')"
+                   required
+                   readonly>
+            <button
+              class="btn btn-small right-gray-border-only background-inherit"
+              ng-click="$ctrl.onDateChange('rangeStart')"
+            >
+             <span class="sr-only">Set range start date</span>
+             <i class="icon-calendar" aria-hidden="true"></i>
+            </button>
+            <input class="form-control input-cursor-pointer"
+                  id="rangeEnd"
+                  name="rangeEnd"
+                  type="text"
+                  ng-value="$ctrl.endDateDisplay"
+                  ng-click="$ctrl.onDateChange('rangeEnd')"
+                  required
+                  readonly>
+            <button
+              class="btn btn-small background-inherit no-border"
+              ng-click="$ctrl.onDateChange('rangeEnd')"
+            >
+             <span class="sr-only">Set range end date</span>
+             <i class="icon-calendar" aria-hidden="true"></i>
+            </button>
+          </div>
+          <p class="color-danger" ng-if="!$ctrl.isValidDateRange">
+            <i>Start and end date range is invalid.</i>
+          </p>
+          <h5 class="modal-content-header-thin">Remove imageries after splitting</h5>
+          <div class="flex-display">
+            <p style="flex: 0.7">
+              <i><span class="color-warning">Warning:</span> Removing imageries from the layer could mess up annotations or analyses, but would prevent duplicated layers in future splitting.</i>
+            </p>
+            <button class="btn select-button btn-tiny" ng-click="$ctrl.onCheckRemoveImages()" style="flex: 0.3">
```

Comment:
I would remove this style tag as well. With the flex-fill class on the `p` it should not be necessary.